aviary/variable_info/functions.py

```diff
@@ -146,12 +146,14 @@ def name_filter(name):
     if overridden_outputs:
         print("\nThe following variables have been overridden:")
         for prom_name in sorted(overridden_outputs):
-            print(f"  '{prom_name}")
+            item = aviary_inputs.get_item(prom_name)
+            print(f"  '{prom_name}  {item[0]}  {item[1]}")
 
     if external_overridden_outputs:
         print("\nThe following variables have been overridden by an external subsystem:")
         for prom_name in sorted(external_overridden_outputs):
-            print(f"  '{prom_name}")
+            item = aviary_inputs.get_item(prom_name)
```

Comment:
The variables in `external_overriden_outputs` are the ones that we have determined are being provided by external subsystems like VSP, so they either won't be in aviary_inputs, or if they are, the value is probably wrong.  I would recommend not printing these ones.

Reply:
Do you mean that I can keep line 149-150 and remove line 155-156 or just withdraw this PR?